src/renderer/components/AccountDistribution/index.js

```diff
@@ -38,31 +36,53 @@ const mapStateToProps = (state, props) => {
   };
 };
 
-class AccountDistribution extends PureComponent<Props, State> {
-  render() {
-    const { accountDistribution } = this.props;
-    return (
-      <>
-        <Box horizontal alignItems="center">
-          <Text ff="Inter|Medium" fontSize={6} color="palette.text.shade100">
-            <Trans
-              i18nKey="accountDistribution.header"
-              values={{ count: accountDistribution.length }}
-              count={accountDistribution.length}
-            />
-          </Text>
-        </Box>
+const AccountDistribution = ({ accountDistribution }: Props) => {
+  const cardRef = useRef(null);
+  const [isVisible, setVisible] = useState(false);
+  useLayoutEffect(() => {
+    const scrollArea = document.getElementById("scroll-area");
+    if (!cardRef.current) {
+      return;
+    }
+    const callback = entries => {
+      if (entries[0] && entries[0].isIntersecting) {
+        setVisible(true);
```

Comment:
i would disconnect observer here aswell since the animation should only occur once :thinking: